src/App.tsx

```diff
@@ -1,96 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { PageInfo } from './components/PageInfo';
+import { Pagination } from './components/Pagination';
+import { PerPageSelector } from './components/PerPageSelector';
+import { ItemsCount } from './types/ItemsCount';
+import { getRangeOfItems } from './utils/helper';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const total = 42;
```

Comment:
would be great to move this total to a separate file with config like you made with ItemsCount. for example `Contstants `file